cmd/daytona/config/const.go

```diff
@@ -86,6 +89,8 @@ func GetScopesFromGitProvider(providerId string) string {
 		return "api,read_user,write_repository"
 	case "bitbucket":
 		return "account:read,repositories:write,pullrequests:read"
+	case "bitbucket-server":
+		return "ACCOUNT_WRITE,PROJECT_VIEW,REPO_READ,REPO_WRITE"
```

Comment:
I think the appropriate scopes should be `PROJECT READ,REPOSITORY_WRITE`

Reply:
REPOSITORY_WRITE is not a valid scope, I had referred scopes for each of the respective functions from [here](https://pkg.go.dev/github.com/gfleury/go-bitbucket-v1#section-readme) (just go to any function definition and required scopes are listed). 
Anyways, Have added couple more scopes, will also give a try for testing the changes myself tomo..

Reply:
i suspect <REPO/PROJECT>_ADMIN scopes are required for authenticated user, so added them..

Reply:
![Screenshot 2024-06-05 at 08 41 49](https://github.com/daytonaio/daytona/assets/26512078/356e8938-dd99-42f7-b298-bd0139293d5c)
![Screenshot 2024-06-05 at 08 41 46](https://github.com/daytonaio/daytona/assets/26512078/cb9e6547-3fe8-4953-acb0-f30103b7143c)

I can not see those scopes in our hosted Bitbucket Server. Also, in the scope descriptions found here https://confluence.atlassian.com/bitbucketserver/http-access-tokens-939515499.html, there are only those I mention above.
![Screenshot 2024-06-05 at 08 42 58](https://github.com/daytonaio/daytona/assets/26512078/3227cdf4-9d7d-4a79-8d85-ce0a3dd824bd)

P.S. Please take a look at https://github.com/daytonaio/daytona/pull/606#issuecomment-2147656758

Reply:
done..